packages/blade/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
For `@typescript-eslint/explicit-function-return-type` I find this rule very aggressive, Sometimes I want typescript to just infer the return type. 
<img width="724" alt="Screenshot 2022-07-25 at 4 23 15 PM" src="https://user-images.githubusercontent.com/35374649/180760844-9e6dfb67-081d-4e58-b5ca-b93dfde140a9.png">

> /* eslint-disable @typescript-eslint/no-shadow */

Seems like a mistake, removed.

Reply:
Yep. Tho I find this rule to be exceptionally useless on JSX attributes. 

because the `CheckboxComponent` already defines the return type of the onChange. Why do I have to explicitly add it here again. Doesn't make sense. 
<img width="614" alt="Screenshot 2022-07-26 at 6 06 18 PM" src="https://user-images.githubusercontent.com/35374649/181007207-64c1ab15-2c19-44a7-b09a-f193170c7fae.png">